lib/modules/manager/gradle/utils.ts

```diff
@@ -11,7 +11,7 @@ const artifactRegex = regEx(
   '^[a-zA-Z][-_a-zA-Z0-9]*(?:\\.[a-zA-Z0-9][-_a-zA-Z0-9]*?)*$',
 );
 
-const versionLikeRegex = regEx('^(?<version>[-_.\\[\\](),a-zA-Z0-9+]+)');
+const versionLikeRegex = regEx('^(?<version>[-_.\\[\\](),a-zA-Z0-9+ ]+)');
```

Comment:
can we make it more strict, so it doesn't allow `[1.6.0, ]  ,  abc` which is invalid obviously.
Let's only allow spaces inside braces 🤔 

This change should be in a separate PR, because it's totally unrelated to the `bump` range strategy

Reply:
How about leaving the version extraction as-is but adding a call to `gradleVersioning.isValid(...)` in `isDependencyString()`?